src/mapblock_mesh.cpp

```diff
@@ -933,7 +933,7 @@ static void updateFastFaceRow(
 		makes_face = next_makes_face;
 		p_corrected = next_p_corrected;
 		face_dir_corrected = next_face_dir_corrected;
-		std::memcpy(lights, next_lights, 4);
+		std::memcpy(lights, next_lights, ARRLEN(lights) * sizeof(u16));
```

Comment:
@sofar the diff you see is a PR diff, not a diff from master. In master we have light[0] = next_lights[0], 1 ,2 ,3 , i just merged the 4 copy to a single memcpy but i did an intermediate error with the content size, it's a u16 not a u8